# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/glibc23/sparc-gdb_fix.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- glibc-2.3.2/sysdeps/unix/sysv/linux/sparc/sys/ucontext.h.orig	2003-08-03 19:45:12.000000000 +0000
+++ glibc-2.3.2/sysdeps/unix/sysv/linux/sparc/sys/ucontext.h	2003-08-03 19:47:29.000000000 +0000
@@ -19,6 +19,8 @@
 #ifndef _SYS_UCONTEXT_H
 #define _SYS_UCONTEXT_H	1
 
+#ifdef __GLIBC__
+
 #include <features.h>
 #include <signal.h>
 #include <bits/wordsize.h>
@@ -267,4 +269,5 @@
   } ucontext_t;
 
 #endif /* __WORDSIZE == 32 */
+#endif /* __GLIBC__ */
 #endif /* sys/ucontext.h */
